hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestPipelineDatanodesIntersection.java

```diff
@@ -127,7 +127,7 @@ public void testPipelineDatanodesIntersection(int nodeCount,
         List<Pipeline> overlapPipelines = RatisPipelineUtils
             .checkPipelineContainSameDatanodes(stateManager, pipeline);
 
-        if (overlapPipelines.isEmpty()) {
```

Comment:
I believe this if condition needs to be fixed, see line 132.

Reply:
However, there is no assertions against any of these data.
We should fix this test probably in another Jira.